index.js

```diff
@@ -67,7 +110,16 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let ratingCount = {}
+
+  for (let i = 0; i < movies.length; i++) {
+    if (ratingCount[movies[i].rated]) {
+      retingCount[movies[i].rated] = 1;
+    }
```

Comment:
So close Kenya!  Right now we are saying if there _is_ a key in ratingCount matching the current movie rating - we need to set the value of ratingCount at that key to be equal to 1.  That is what we want to do if there is _not_ a value in ratingCount.

Take another attempt and see if you can crack it!